ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh

```diff
@@ -4,19 +4,15 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-if test ! -f "${KUBECONFIG}"
-then
-	echo "No kubeconfig, so no point in gathering loki logs."
-	exit 0
-fi
+export LOKI_VERSION="1.3.0"
 
-cat << EOF > /tmp/loki-manifests
----
+cat >> "${SHARED_DIR}/manifest_loki-ns.yml" << EOF
 apiVersion: v1
 kind: Namespace
 metadata:
   name: loki
----
+EOF
+cat >> "${SHARED_DIR}/manifest_loki-clusterrole.yml" << EOF
```

Comment:
I am missing the manifests moved into `${SHARED_DIR}` in this PR. Are they already there?

Reply:
Other steps may create additional manifests (i.e. [persistent metrics](https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/monitoringpvc/ipi-install-monitoringpvc-commands.sh))